models/public/ssd300/ssd300.md

```diff
@@ -58,25 +66,25 @@ Channel order is `BGR`.
 
 ### Original model
 
-The array of detection summary info, name - `detection_out`,  shape - `1, 1, N, 7`, where N is the number of detected bounding boxes. For each detection, the description has the format:
+The array of detection summary info, name - `detection_out`, shape - `1, 1, N, 7`, where N is the number of detected bounding boxes. For each detection, the description has the format:
 [`image_id`, `label`, `conf`, `x_min`, `y_min`, `x_max`, `y_max`], where:
 
 - `image_id` - ID of the image in the batch
-- `label` - predicted class ID
-- `conf` - confidence for the predicted class
+- `label` - predicted class ID (0 - background, 1..20 - PASCAL VOC defined class ids)
```

Comment:
backgroud is supressed by detection output, so I think it should be skipped for end user
```suggestion
- `label` - predicted class ID (1..20 - PASCAL VOC defined class ids)
```